longoverdue.py

```diff
@@ -170,44 +177,54 @@ def restart():
     userservices = set()
     services = set()
     for proc in procs:
-        if proc.uunit != "-":
-            if proc.uunit.endswith(".service"):
-                userservices.add(proc.uunit)
-        elif proc.unit.endswith(".service"):
-            if proc.unit not in NO_AUTORESTART:
-                services.add(proc.unit)
+        if proc:
```

Comment:
Do you have an example of a situation that can cause this?

Reply:
That just happened on one of my servers. I'm not really sure either, but since that fixed it nicely I wasn't going to get deep into `getprocs`. Maybe it can be fixed 'cleaner' there. But that is not scope if this PR.

Reply:
It's always good to understand why the change is being made. When submitting a bug fix, you should verify that it is correct. Even if the error message goes away, you haven't necessarily fixed the underlying issue. Maybe something else went wrong and it would be better to warn about it? Maybe the code is just incorrect and silently ignoring the problem results in missing information in the output? This is definitely within scope.

Looking at the code, I don't see how `proc` itself can be falsy at all. The traceback you provided points to `proc.uunit` being `None` instead. This pushes the scale towards "lazy workaround" rather than a proper fix. I can't merge this without further investigation.

Reply:
Proc itself must be None, otherwise the workaround would not work at all.

Look, I don't want to be that guy that opens a PR and doesn't respond to reviews, certainly not, but for such a simple tool I just don't have the time to debug it further. Its your code at the end of the day. I'm at 'take it or leave it' at this point for this particular change.